src/api/db/data/20200319095810_remove_old_rss_notifications.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveOldRssNotifications < ActiveRecord::Migration[5.2]
+  def up
+    Notification.where(notifiable_id: nil).or(Notification.where(notifiable_type: nil)).delete_all
```

Comment:
Hm, this is time sensitiive right? If I run this in the next release I will not have any `Notifications` anymore

Reply:
Upgrading to the next release will delete all notification records. But new notification records will be created thanks to the code changes made in pull requests, previously merged (#9243 and friends). So yes, you will not have any  _old_ `Notifications`, but you will receive notifications from events that happen since  the upgrade, and later on.

Reply:
I don't think you should do that. You should handle the `Notification`s that are there and that are relevant.

Reply:
What do you mean by _handle_ and _relevant_?

Reply:
* handle == set the notifiable
* relevant == newer than 100 days

Reply:
This would mean to create a migration of the notifications newer than 100 days, for users that have logged in the last 100 days and that have their RSS feed enabled. And they would be 10 notification registers maximum per user. Keeping in mind that emails woin't be affected, and only users that use RSS feeds, worst case scenario is that the users won't receive last 10 notifications through their RSS feeds. I don't think the work invested in doing this migration is worth it. Is this really needed?

Reply:
>  worst case scenario is that the users won't receive last 10 notifications through their RSS feeds

Often people use RSS feeds for automation, yes I think this would not be nice. Doesn't have to be in this PR though...

Reply:
Ok, I will modify this pull request, and I will substitute the deletion of data with a migration. Marking this pull request as WIP till this is done.

Reply:
We discussed offline and decided not to run a migration in this pull request. So for rss feeds, data will be read from the old `event_payload` field only when it is necessary.